docs/HowToGuides/FirstPullRequest.md

```diff
@@ -0,0 +1,169 @@
+# How to Submit Your First Pull Request
+
+So you've decided to contribute to the Swift toolchain, welcome!
+Maybe this is your first time contributing to an open source project, or maybe
+you are an experienced open source contributor who is excited about Swift, or
+maybe you are somewhere in-between. Regardless of your background, we are
+excited to have you contribute and improve the developer experience for Swift
+programmers all over the globe.
+:sparkles::child::student::woman_technologist::technologist::man_technologist::sparkles:
+
+This document provides a high-level overview of different parts of the
+contribution process.
+
+## How do I pick something to work on?
+
+In case you don't have something specific you'd like to work on, such as
+implementing something for a Swift Evolution pitch, you could start off by
+working on a bug labeled `StarterBug` on [Swift JIRA][StarterBug]. If the
+bug hasn't been assigned to someone, check the comments in case someone has
+already started working on it. If not, feel free to assign it to yourself and
+start working on it!
+
+[StarterBug]: https://bugs.swift.org/issues/?jql=labels%20%3D%20StarterBug%20AND%20(status%20%3D%20Open%20OR%20status%20%3D%20Reopened)%20AND%20project%20%3D%20Swift 
+
+## Getting Help
+
+Usually, Starter Bugs try to provide some instructions to help you get started.
+In case those are missing, please ask the bug reporter for more detailed steps
+and they will be happy to help.
+
+Once you start working on the bug, you will inevitably end up having a lot of
+questions. Don't be afraid to ask for help! The codebase is large and wrapping
+your head around it will take time. For example, you might have questions like:
```

Comment:
Somewhere near or before this point, we should address the Code of Conduct and, more generally, communicating kindly and constructively. (I started playing with some wording yesterday, but got sidetracked on other issues.)

Reply:
We talk about the Code of Conduct right in the README, before you even get to the First Pull Request guide. Do you think that's sufficient or that we should re-iterate it here?

<img width="750" alt="Screen Shot 2020-09-04 at 11 25 30 AM" src="https://user-images.githubusercontent.com/52841859/92274074-6cc1b180-eea1-11ea-99d3-2d493a425278.png">

Reply:
I could easily see someone hitting this document first, and it never hurts to emphasize the seriousness of the CoC. Maybe we can get away with a quick mention at the end of the "Getting Help" section, though—something like:

```markdown
Remember that the [Swift Code of Conduct](...) applies whenever you are participating
in the Swift project. See [the readme file](...) for more on this.
```